examples/dynamic-enums-example.yaml

```diff
@@ -0,0 +1,140 @@
+  bioregistry: https://bioregistry.io/registry/
+  MONDO: http://purl.obolibrary.org/obo/MONDO_
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  loinc: http://loinc.org/
+
+default_prefix: ex
+default_range: string
+
+default_curi_maps:
+    - semweb_context
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - owl
+
+imports:
+  - linkml:types
+
+  
+#==================================
+# Classes                         #
+#==================================
+
+classes:
+  HumanSample:
+    slots:
+      - name
+      - disease
+
+
+#==================================
+# Slots                           #
+#==================================
+
+slots:
+  name:
+    range: string
+  disease:
+    range: HumanDisease
+  vital_status:
+    enum_range:
+      permissible_values:
+        LIVING:
+        DEAD:
+        UNDEAD:
+
+#==================================
+# Enums
+#==================================
+
+enums:
+  Disease:
+    reachable_from:
+      source_ontology: bioregistry:mondo
+      source_nodes:
+        - MONDO:0000001 ## disease or disorder
+      is_direct: false
+      relationship_types:
+        - rdfs:subClassOf
+    minus:
+      permissible_values:
+        root_node:
+          meaning: MONDO:0000001 ## disease or disorder
+  HumanDisease:
+    description: Extends the Disease value set, including NCIT neoplasms, excluding non-human diseases
+    inherits:
+      - Disease
+    include:
+      - reachable_from:
+          source_ontology: bioregistry:ncit
+          source_nodes:
+            - NCIT:C3262
+    minus:
+      - reachable_from:
+          source_ontology: bioregistry:mondo
+          source_nodes:
+            - MONDO:0005583 ## non-human animal disease
+          relationship_types:
+            - rdfs:subClassOf
+      - permissible_values:
+          NOT_THIS_ONE:
+            meaning: MONDO:9999
+            description: Example of excluding a single node
+
+      
+  LoincExample:
+    enum_uri: http://hl7.org/fhir/ValueSet/example-intensional
+    see_also:
+      - https://build.fhir.org/valueset-example-intensional.json.html
+    include:
+      - reachable_from:
+          source_ontology: bioregistry:loinc
+          source_nodes:
+            - loinc:LP43571-6
+          is_direct: true
+    minus:
+      concepts:
+        - LOINC:5932-9
+
+  HCAExample:
+    see_also:
+      - https://github.com/linkml/linkml/issues/274
+    include:
+      - reachable_from:
+          source_ontology: bioregistry:go
+          source_nodes:
+            - GO:0007049
+            - GO:0022403
+          include_self: false
+          relationship_types:
+            - rdfs:subClassOf
+    minus:
+      concepts:
+        - LOINC:5932-9
+        
+  Brand:
+    enum_uri: wikidata:Q431289
+    include:
+      - reachable_from:
+          source_ontology: bioregistry:wikidata
+          source_nodes:
+          - wikidata:Q431289
+          include_self: false
+          relationship_types:
```

Comment:
`relationship_types`: Would like a definition for this field.

I do understand what a relationship type is, but I'm just wondering how listing them out in the enum definition is helpful / how intends to exactly be used.